diff --git a/live2dEasyControl.js b/live2dEasyControl.js
index 80a5355da5ef7381c57fa077d6aef2c208c7602c..be219dfdf6031cecb4f71c079ec4fd9dea64854c 100644
--- a/live2dEasyControl.js
+++ b/live2dEasyControl.js	
@@ -2885,12 +2885,13 @@ class Os {
   createTextureFromPngFile(t, e, i) {
     for (let a = this._textures.begin(); a.notEqual(this._textures.end()); a.preIncrement())
       if (a.ptr().fileName == t && a.ptr().usePremultply == e) {
-        a.ptr().img = new Image(), a.ptr().img.addEventListener("load", () => i(a.ptr()), {
+        a.ptr().img = new Image(), a.ptr().img.crossOrigin = "anonymous", a.ptr().img.addEventListener("load", () => i(a.ptr()), {
           passive: !0
         }), a.ptr().img.src = t;
         return;
       }
     const s = new Image();
+    s.crossOrigin = "anonymous";
     s.addEventListener(
       "load",
       () => {
